conf/messages.en

```diff
@@ -185,14 +185,18 @@ freeHoursResult.partialEligible.guidance.england = <<Need to get content and eli
 freeHoursResult.partialEligible.guidance.scotland = On this scheme, you can get free hours for your 3 or 4 year old that can be used in term time - this is 600 hours a year. You can take up the offer the term following your child’s 3rd birthday. Your provider may apply charges for additional services.
 freeHoursResult.partialEligible.guidance.wales = On this scheme, you can get free hours for your 3 or 4 year old that can be used in term time - this is 390 hours a year. You can take up the offer the term following your child’s 3rd birthday. Your provider may apply charges for additional services.
 freeHoursResult.partialEligible.guidance.northern-ireland = On this scheme, you can get free hours of free pre-school education for your 3 or 4 year old that can be used in term time - this is 475 hours a year. You can take up the offer the term following your child’s 3rd birthday. Your provider may apply charges for additional services.
-freeHoursResult.toBeEligible.heading = If you want to see what support you could get
+freeHoursResult.toBeEligible.heading = See what help you could get
 freeHoursResult.toBeEligible.info1.start = You‘re currently not eligible for any free hours because you don’t have a child aged 3 or 4.
 freeHoursResult.toBeEligible.info1.link.text = answer as though your 2 year old is aged 3 or 4
 freeHoursResult.toBeEligible.info2 = You also need to have childcare costs for the other available schemes. So, if you’re here to see what childcare you can afford, you should tell the calculator that you expect to have childcare costs. You’ll need to give an estimate of how much you think you might pay as part of the calculation.
-freeHoursResult.toBeEligible.info2.start  = To see what support you could get, you should select
+freeHoursResult.toBeEligible.info2.start  = As you need to have childcare costs, you should select
 freeHoursResult.toBeEligible.info2.end = You’ll need to give an estimate of how much you think you might pay for childare.
 freeHoursResult.toBeEligible.info2.link.text = not yet but maybe in the future.
 
+freeHoursResult.toBeEligible.info2b.start  = As you need to have approved childcare, you should select
+freeHoursResult.toBeEligible.info2b.end = You’ll need to give an estimate of how much you think you might pay for childare.
```

Comment:
Can we please fix the typo on childcare? Could we also change the same typo on element "freeHoursResult.toBeEligible.info2.end". Thanks.

Reply:
Should be fixed now... sorry didnt get a notification about this